public/js/app/templates/timelinePostTemplate.handlebars

```diff
@@ -7,14 +7,13 @@
     <div class="title">
       {{view 'post-title' post=post.model}}
     </div>
-
```

Comment:
sorry for nitpicking, but this line added a bit of readability :)
can you restore it please?